src/www/ui/api/index.php

```diff
@@ -269,6 +269,7 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
     $app->get('', LicenseController::class . ':getAllLicenses');
     $app->post('/import-csv', LicenseController::class . ':handleImportLicense');
     $app->post('', LicenseController::class . ':createLicense');
+    $app->put('/merge/{shortname:[\\w\\- \\.]+}', LicenseController::class . ':mergeLicense');
```

Comment:
As requested, please use following regex for license shortname:
```suggestion
    $app->put('/merge/{shortname:.+}', LicenseController::class . ':mergeLicense');
```

Reply:
Done